### ISSUE TITLE
Icon and name is too similar to official Godot engine

### ISSUE THREAD

**Issue by bbhtt** [flathub/io.github.MakovWait.Godots#5]
Hello we are tracking a bunch of applications with trademark violations or icons that are copies of other application icons.

The current icon and name used by the app shares too many elements from the official Godotengine published on Flathub (unofficially) while this is no way related to that.

Please try to use your own icon for the application to avoid trademark issues and confusing people. I think 2 months is a reasonable time period for us to wait for the icon and name to be changed.

You can find some resources for icon guidelines in https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#app-icon and name in https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#app-name

And the guidelines on trademarks https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#no-trademark-violations

**Comment by coffandro** [comment]
This isn't the actual project, it is found at https://github.com/MakovWait/godots so that's where it would need to be changed. 
besides that at the same time this dosen't actually violate *any* trademarks since if Godot's trademark is open to all contributers, which from what I can see includes extension developers like the Godots developer.

**Comment by bbhtt** [comment]
>  since if Godot's trademark is open to all contributers, which from what I can see includes extension developers like the Godots developer.

Can you please point me to some link for that?

If no trademark is violated then it is probably as the logo is already a slight modification.

**Comment by coffandro** [comment]
Here you go: https://godotengine.org/governance/

**Comment by bbhtt** [comment]
Where does it say that? I see it saying that the Foundation holds trademarks

> While the Godot Foundation holds assets on behalf of Godot including trademarks

only the copyright to the code is held by contributors:

> the copyright to Godot's source is held collectively by every contributor.

**Comment by coffandro** [comment]
Oh wait yeah I misread, anyway this is still an upstream issue and at the same time this repo isn't really maintained.

**Comment by bbhtt** [comment]
cc-in upstream then, @MakovWait This affects the listing on Flathub.

**Comment by coffandro** [comment]
This repo seems abandoned tbh

**Comment by bbhtt** [comment]
what's the update here?

**Comment by coffandro** [comment]
That repo is abandoned :/

**Comment by GreatNovaDragon** [comment]
It looks like the creator just waited on Godot 4.3, and a PR just got pushed in.

**Comment by AsciiWolf** [comment]
The app logo looks fine in my opinion and clearly represents what the app does. It should be considered as "fair use" in my opinion. However, I am not a lawyer.

By the way, I would be more worried about [godotGem](https://flathub.org/apps/io.github.iSlammedMyKindle.godotGem) that seems to be an app completely unrelated to Godot while using a desktop icon very similar to the Godot one (probably just a modified Godot icon) and also having "godot" in its name.

**Comment by coffandro** [comment]
Yeah, godotGem is just a frontend for ViGEm built in Godot, where Godots is a manager of godot versions
